src/Conda.jl

```diff
@@ -224,6 +224,17 @@ function list(env::Environment=ROOTENV)
     runconda(`list`, env)
 end
 
+"""
+List all packages and write them to an export file for use the Conda.create
+NOTE: Because export is a reserved word we use the term freeze from python
+"""
+function freeze(filepath::AbstractString, env::Environment=ROOTENV)
+    _install_conda(env)
+    open(filepath, "w") do fobj
+        write(fobj, read(_set_conda_env(`$conda list --export`, env), String))
```

Comment:
I think you can just use `read` instead of `read(..., String)` here.

Reply:
Oh, yeah, I guess we're just dumping the output directly to a file anyways.